components/dependency_injection.rst

```diff
@@ -302,6 +302,25 @@ config files:
             ->register('newsletter_manager', 'NewsletterManager')
             ->addMethodCall('setMailer', [new Reference('mailer')]);
 
+    .. code-block:: php-fluent-di
+
+        use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+        use function Symfony\Component\DependencyInjection\Loader\Configurator\ref;
+
+        return function(ContainerConfigurator $configurator) {
+            $configurator->parameters()
+                ->set('mailer.transport', 'sendmail');
+
+            $container = $configurator->services();
+
+            $container->set('mailer', 'Mailer')
+                ->args(['%mailer.transport%']);
+
+            $container->set('newsletter_manager', 'NewsletterManager')
+                ->call('setMailer', [ref('mailer')]);
+        };
```

Comment:
What about showing an alias too?

Reply:
up to you boss, i'll do whatever. I tried to keep the examples exactly the same as the yaml/xml counterparts to be consistent.

Reply:
I think this is a basic and deserves an addition in all example :). What do others think?

Reply:
I updated every place that references di configuration, and we do have several examples that show using aliases. I think if we *were* to update the main examples to showcase aliases, that should be done in a separate as a separate change.